geopandas/base.py

```diff
@@ -843,6 +843,36 @@ def has_z(self):
         """
         return _delegate_property("has_z", self)
 
+    @property
+    def has_m(self):
+        """Returns a ``Series`` of ``dtype('bool')`` with value ``True`` for
+        features that have a m-component.
+
+
```

Comment:
Could we add something to the docstring this this requires shapely 2.1+? (which applies to all new methods from shapely 2.1 that we are adding)